homeassistant/const.py

```diff
@@ -740,6 +740,21 @@ class UnitOfPower(StrEnum):
     UnitOfPower.KILO_WATT,
     "2025.1",
 )
+"""Deprecated: please use UnitOfPower.MEGA_WATT."""
+_DEPRECATED_POWER_MEGA_WATT: Final = DeprecatedConstantEnum(
+    UnitOfPower.MEGA_WATT,
+    "2025.1",
+)
```

Comment:
You don't need to mark them as deprecated if they didn't exist before...
You need to adjust the tests instead